.github/dependabot.yml

```diff
@@ -8,19 +8,7 @@ updates:
     allow:
       - dependency-type: "all"
     open-pull-requests-limit: 100
-  - package-ecosystem: "cargo"
-    directory: "/tests/host_tools/uffd/"
-    schedule:
-      interval: "weekly"
-      day: "monday"
-    allow:
-      - dependency-type: "all"
-    open-pull-requests-limit: 100
-  - package-ecosystem: "cargo"
-    directory: "/tests/integration_tests/security/demo_seccomp/"
-    schedule:
-      interval: "weekly"
-      day: "monday"
-    allow:
-      - dependency-type: "all"
-    open-pull-requests-limit: 100
+    groups:
```

Comment:
just for my education, do we have to explicitly say we're using a beta somewhere in the config?

Reply:
Mh, I couldnt find anything on that in the docs, so I don't think so? https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups